ServiceWebsite/ServiceWebsite.AcceptanceTests/Constants/RepresentativePageNames.cs

```diff
@@ -6,14 +6,9 @@ namespace ServiceWebsite.AcceptanceTests.Constants
 {
     public class RepresentativePageNames
     {
-        public const string AboutYou = "about you";
-        public const string AboutYourClient = "about your client";
-        public const string AccessToRoom = "access to a suitable room";
-        public const string ClientAttendance = "client attendance";
-        public const string HearingSuitability = "hearing suitability";
-        public const string YourComputer = "your computer";
-        public const string AboutYourComputer = "about your computer";
-
+        public const string AppointingABarrister = "appointing a barrister";
+        public const string OtherInformation = "other information";
+        public const string AnswersSaved = "answers saved";
         public const string QuestionnaireCompleted = "questionnaire completed";
```

Comment:
I don't think we need these pages here. Please check
QuestionnaireCompleted 
ContactUs
Thankyou